src/components/faktum/Faktum.tsx

```diff
@@ -48,117 +62,112 @@ export function Faktum(props: IFaktum<QuizFaktum | IQuizGeneratorFaktum>) {
 
   function renderFaktumType() {
     if (faktum.beskrivendeId === FAKTUM_GAARDSBRUK_ARBAAR_FOR_TIMER) {
+      if (faktum.readOnly || readonly) {
+        return (
+          <FaktumNumberReadOnly
+            faktum={faktum as IQuizNumberFaktum}
+            showAllFaktumTexts={showAllFaktumTexts}
+          />
+        );
+      }
+
       return (
         <FaktumEgetGaardsbrukArbeidsaar
           ref={faktumRef}
           faktum={faktum as IQuizNumberFaktum}
           onChange={props.onChange}
-          readonly={readonly}
         />
       );
     }
 
     switch (faktum.type) {
       case "boolean":
-        return (
-          <FaktumBoolean
-            ref={faktumRef}
-            faktum={faktum}
-            onChange={props.onChange}
-            readonly={readonly}
-          />
-        );
+        if (faktum.readOnly || readonly) {
```

Comment:
Kanskje sette denne logikken ut i en egen variabel? Eksempelvis `const showReadOnly = faktum.readOnly || readonly`. Siden den repeteres så mange steder.

Reply:
Når det bare to boolske verdier så synes jeg det er bedre å bare ha det inline. Hadde det vært flere så hadde jeg vært enig :)